drivers/input/i8042prt/keyboard.c

```diff
@@ -369,7 +369,7 @@ i8042KbdDpcRoutine(
 		DeviceExtension->KeyboardBuffer + KeysInBufferCopy,
 		&KeysTransferred);
 
```

Comment:
Could you also `ASSERT(KeGetCurrentIrql() == Irql)` here please to validate that the callback didn't change it? Thanks.

Reply:
Done. (LiveCD still works fine, as expected.)

NB: This file has a mix of tabs/spaces. I kept tabs here...